src/utils/setup.mjs

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: GSConnect Developers https://github.com/GSConnect
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+import GLib from 'gi://GLib';
+
+const [filename] = GLib.filename_from_uri(import.meta.url);
+const dirname = GLib.path_get_dirname(filename);
+imports.searchPath.unshift(dirname);
+
+export const {setup, setupGettext} = imports.setup;
+export default setup;
```

Comment:
OOC, why does `setup.mjs` here export `setup`, but `remote.mjs` (not in this PR) export `Remote`?

Does the difference in casing have any conceptual significance? Can they be used interchangeably (case-insensitively), or does it make a practical difference? Should we settle on a standard capitalization style?

Reply:
`setup` is a function name, hence it's exported as lowercase, whereas `Remote` is a namespace, hence capitalised. This is essentially the same naming convention I see throughout the rest of the extension and most JavaScript code in general.

Since it's a default export, `Remote` isn't a name exposed to modules that import it anyway, e.g. `import remote` or `import Remote from './remote.mjs'` are both valid, dynamic import works as `(await import('./remote.mjs')).default`

IIRC, in #1683 I export the members directly since the bridge is no longer required